src/swedbankpay.js

```diff
@@ -8,6 +8,12 @@ import "prismjs/themes/prism.css";
 import "./less/swedbankpay.less";
 import "./less/documentation-swedbankpay.less";
 
+import "@src/img/swedbankpay/logo/swedbankpay-logo-neg.svg";
```

Comment:
you can remove this one, no?

Reply:
Yeah, forgot about that. Will be removed!